launchpod/src/main/webapp/angular-launchpod/src/app/app.module.ts

```diff
@@ -32,7 +35,8 @@ import { MatListModule } from '@angular/material/list';
     MatButtonModule,
     MatSidenavModule,
     MatIconModule,
-    MatListModule
+    MatListModule,
+    ClipboardModule
```

Comment:
since the imports are alphabetized, you should move the clipboard module under browsermodule

Reply:
done.